repo-tools/scripts/synchronize-yaml.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
make sure you get the "brew" bash on macOS so it isn't ancient bash:
```suggestion
#!/usr/bin/env bash
```